cmd/rpcdaemon/commands/coinbase.go

```diff
@@ -2,11 +2,16 @@ package commands
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/ledgerwatch/turbo-geth/common"
 )
 
-// Etherbase is the address that mining rewards will be send to
+// Coinbase is the address that mining rewards will be sent to
 func (api *APIImpl) Coinbase(_ context.Context) (common.Address, error) {
+	if api.ethBackend == nil {
+		// We're running in --chaindata mode or otherwise cannot get the backend
+		return common.Address{}, fmt.Errorf("eth_coinbase function is not available")
```

Comment:
Could you add some extra text here, so that it reads "eth_coinbase function is not available, please use --private.api.addr option instead of --chaindata option", so that it is clear that the function can work, but different options

Reply:
You can also introduce a string constant and use `fmt.Errorf("%s %s", "eth_coinbase", BlahConstant)`

Reply:
Or `fmt.Errorf("eth_coinbase %s", BlahConstant)` :)

Reply:
I created a file called `error_message.go` in `./cmd/rpcdaemon/commands` where we can collect error messages. Makes it easier to play around with messages later.

Reply:
Great, I love it